tests/e2e/testsuites/specs.go

```diff
@@ -41,6 +41,7 @@ type VolumeDetails struct {
 	VolumeBindingMode     *storagev1.VolumeBindingMode
 	AllowedTopologyValues []string
 	VolumeMount           VolumeMountDetails
```

Comment:
Propose to add a new field `VolumeMode string` with 
```
type VolumeMode int
var (
Block VolumeMode = int
FileSystem VolumeMode
)
```
And use the field for following checks in other place to avoid confusion/buggy cases like both VolumeMount and VolumeBlock are set

Reply:
Great idea.